### ISSUE TITLE
Broken PDF using html tag small in acrobat reader

### ISSUE THREAD

**Issue by nck974** [Kozea/WeasyPrint#1351]
Example to reproduce:
```html
<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Report</title>
    <meta name="description" content="Report example">
  </head>

  <body>
      <small>
        Sometext
      </small>
  </body>
</html>
```
Opening with acrobat reader produces `There was a problem... (117)`

**Comment by grewn0uille** [comment]
Hello!

Just tried with the current `master` and `52.x` and I didn’t reproduce.

Can you tell us which version of WeasyPrint do you use and attach the PDF you got please?

**Comment by nck974** [comment]
Hello, I'm using 52.4 installed from PIP, I just attached the file which I created from the exact html I posted using;
```
python -m weasyprint .\delete_me.html .\delete_me.pdf
```
[delete_me.pdf](https://github.com/Kozea/WeasyPrint/files/6434456/delete_me.pdf)

**Comment by grewn0uille** [comment]
By looking the font included in the PDF, it’s the same problem as #1292.

It’s caused by a bug in Cairo. You can downgrade it (version 1.17.2 is working).
